src/main/java/sx/blah/discord/modules/ModuleLoader.java

```diff
@@ -280,25 +304,25 @@ public static void loadExternalModules(List<File> files) {
 							Class.forName(clazz);
 							loaded = true;
 						} catch (ClassNotFoundException ignored) {}
-						
+
 						if (!loaded)
 							loaded = findFileForClass(files, clazz) != null;
-						
+
 						if (!loaded)
 							break;
 					}
 				} catch (IOException ignored) {}
-				
+
 				if (loaded)
 					loadExternalModules(file);
-				
+
 				return loaded;
 			}));
-			
+
 			if (dependents.size() == 0)
 				break;
 		}
-		
+
```

Comment:
I am super sorry about this weirdness. I even tried manually typing in the tabs to get it to stop it showing up as a change but it wouldn't take them :-(